tracer/test/Datadog.Trace.Security.IntegrationTests/AspNetBase.cs

```diff
@@ -117,19 +117,72 @@ public async Task TestBlockedRequestAsync(MockTracerAgent agent, string url, int
 
                 return target.Tags;
             }));
-
             // Overriding the type name here as we have multiple test classes in the file
             // Ensures that we get nice file nesting in Solution Explorer
             await Verifier.Verify(spans, settings)
                           .UseMethodName("_")
                           .UseTypeName(GetTestName());
         }
 
-        protected void SetHttpPort(int httpPort)
+        protected async Task<System.Collections.Immutable.IImmutableList<MockSpan>> SendRequestsAsync(MockTracerAgent agent, string url, int expectedSpans, int attackNumber, bool parallel = false)
```

Comment:
[nit] can we add the namespace in the using at the top instead (for readability)?

Reply:
fixed in c0b8d65339256d683ac052d315567275b99859fa